### DISCUSSION TITLE
Akash Analytics (draft)

### DISCUSSION THREAD

**Discussion by zackgow** [akash-network/community#234] [Akash Delegation Framework]

**Comment by anilmurty** [comment]
Hey @zackgow - thanks so much for this proposal. I realize this is a draft and will be fleshed out with details but we appreciate you putting this together and also appreciate it being open source. We plan to discuss this (along with any other active proposals and projects) during out monthly Steering Committee meeting, happening in a couple hours from now: https://github.com/akash-network/community/tree/main/committee-steering. -- please join that if you are able to

**Comment by bmenzalji** [comment]
Hey @zackgow are both the projects you referenced live? 

https://secretanalytics.xyz/ - 500 internal server error
https://jackal-dashboard-2sizcg3ipa-ue.a.run.app/ - 404 page not found

Can you please reference the updated links?

**Comment by bmenzalji** [comment]
Changed category from governance to delegation since this proposal is asking for delegations.

**Comment by bmenzalji** [comment]
@zackgow can you DM me on Discord?  Have a few questions for delegations.  Boz_M in the akash discord server.

**Comment by brewsterdrinkwater** [comment]
Cloudmos indexer is open source. 

You can find realtime data at stats.akash.network
